README.md

```diff
@@ -7,11 +7,26 @@ For the documentation see: http://metafacture.github.io/metafacture-documentatio
 
 This repo is the central place for the documentation about Metafacture that we render using the jekyll template [Just the docs](https://github.com/just-the-docs/just-the-docs).
 
-The content pages can be found [here in `/docs/`](/docs/)
-
+The content pages can be found [in `/docs/`](/docs/)
+
+## Installation
+If you are on debian derivates go install build-essentials:
+```
+ apt install build-essential
```

Comment:
Is the whitespace intentional?

Reply:
Yeah I thought so because here it is also one prefixing whitespace: https://github.com/metafacture/metafacture-documentation/blob/master/docs/Documentation-Maintainer-Guide.md?plain=1#L33C1-L34C2 , but at other places it is not. We can or should remove all prefixing whitespaces?

Reply:
I've removed these whitespaces here.